src/main/java/UI/ProfilePage.java

```diff
@@ -1,89 +1,123 @@
-    //    this.rating = user.calculateRating(); // Uses the implementation in User class.
-    //    this.profilePic = profilePic;
-    //}
-//
-    ///**
-    // * The ProfilePage constructor with no profilePic, so their picture will be a default picture and profilePic is
-    // * null.
-    // * @param title The title of the profile page.
-    // * @param user The current user of the program.
-    // */
-    //public ProfilePage(String title, User user) {
-    //    super(title);
-    //    this.user = user;
-    //    this.username = user.getUsername();
-    //    this.email = user.getEmail();
-    //    this.reviews = user.getReviews();
-    //    this.rating = user.calculateRating(); // Uses the implementation in User class.
-    //    this.profilePic = null;
-    //}
-//
-    //// Note there are no setters for username, email, reviews, or rating since they all depend on user and
-    //// shouldn't be changed.
-    //public User getUser() {
-    //    return this.user;
-    //}
-//
-    //public String getUsername() {
-    //    return this.username;
-    //}
-//
-    //public String getEmail() {
-    //    return this.email;
-    //}
-//
-    //public List<Review> getReviews() {
-    //    return this.reviews;
-    //}
-//
-    //public int getRating() {
-    //    return this.rating;
-    //}
-    //public String getProfilePic() {
-    //    return this.profilePic;
-    //}
-//
-    //public void setProfilePic(String imagePath) {
-    //    this.profilePic = imagePath;
-    //}
-//
-    ///**
-    // * TODO
-    // * This method displays all of the pertinent information to the profile page that we collected from the current
-    // * user object when the profile button is clicked. I'm not sure what we are doing for clean architecture,
-    // * but I think this would be in some kind of controller class for our program.
-    // */
-    //public void displayProfile() {
-//
-    //}
-//
-}
 
+    /**
+     * This sets up the ProfilePage GUI by creating all the necessary JLabels, JButtons, and JPanels and
+     * adding them to the ProfilePage.
+     */
+    private void setUpPanel() {
+        this.setPreferredSize(new Dimension(WIDTH, HEIGHT));
+        this.setLayout(LAYOUT);
+
+        titleLabel = new JLabel(PAGETITLE);
+        titleLabel.setFont(new Font("Arial", Font.BOLD, 40));
+        this.add(titleLabel);
+
+        usernameLabel = new JLabel("Username: " + username);
+        usernameLabel.setFont(new Font("Arial", Font.PLAIN, 20));
+        this.add(usernameLabel);
+
+        emailLabel = new JLabel("Email: " + email);
+        emailLabel.setFont(new Font("Arial", Font.PLAIN, 20));
+        this.add(emailLabel);
+
+        ratingLabel = new JLabel("Rating: " + rating);
+        ratingLabel.setFont(new Font("Arial", Font.PLAIN, 20));
+        this.add(ratingLabel);
+
+        reviewForm = new ProfileReviewPanel();
+        this.add(reviewForm);
+
+        ImageIcon imageIcon = new ImageIcon("images/profile_picture.png");
```

Comment:
I heard from Luke that you might have wanted to let the user upload their own profile image?
If that is the case, feel free to take a look at my implementation
https://github.com/CSC207-2022F-UofT/course-project-scamazon-ca-group-118/blob/8267d878ae8ae7c430327bd1dd4ba34b75131fdd/src/main/java/UI/CreateListingPage.java#L106

If you have any questions about it or need help, feel free to message me.